### DISCUSSION TITLE
Status and future of visx

### DISCUSSION THREAD

**Discussion by jimmy-e** [airbnb/visx#1394] [Q&A] [answered]
Hi @williaster , I am wondering what the status is of this project. Me and my team are interested in using this project, however I have noticed that there has not been a release since March 8th and recently a lot of issues are going unanswered. Is there going to be continued support of visx by the airbnb team? Thanks!

**Comment by williaster** [comment, marked as answer]
Hi, they are not marked as releases on GitHub because of the way we automated it, but we have released dozens and dozens of versions this year. https://github.com/airbnb/visx/blob/master/CHANGELOG.md

I'm on leave for all of Dec and am the primary maintainer, hence some open issues are building up. I appreciate everyone understanding that work breaks are important :)

**Reply by jimmy-e** [reply]
Thanks Chris and apologies for my confusion. I hope you have a great holidays!
